pypeit/specobjs.py

```diff
@@ -608,6 +608,12 @@ def __getattr__(self, k):
         # Recast as an array
         return lst_to_array(lst)
 
+    def __getstate__(self):
```

Comment:
I'm not following why these changes were needed?

Reply:
I needed to save the SpecObjs into a pickle file to be stored in the dev suite, and without these being set, I ran into a recursion problem.

Reply:
I don't think we need this function anymore, since now interactive will just create manual parameters right? Maybe it does not hurt to have it though, but ask @profxj who has reworked specobj.

Reply:
Agreed - removed.